docs/dependency_confusion.md

```diff
@@ -1,140 +1,55 @@
-dependency attacks.
+### npm
 
-## *[Cachito][cachito-1] also supports the following package managers, but Cachi2 does not (yet).*
+- package manager for JavaScript
+- lock file: `package-lock.json`
+- [docs/npm.md](https://github.com/containerbuildsystem/cachi2/tree/main/docs/npm.md)
 
-> ## pip
->
-> *TL;DR: if your package is Cachito compliant, it is most likely safe. If you want to be sure, verify
-> checksums.*
->
-> Python packages can have various formats, which are specified across multiple PEPs. Pip supports
-> most of them. Cachito support for pip, on the other hand, is quite simple. You must define all the
-> direct and indirect dependencies of your package in
-> [requirements.txt](https://pip.pypa.io/en/stable/user_guide/#requirements-files) style files and
-> tell Cachito to process them.
->
-> Cachito further restricts what you can put in your requirements.txt files. All of the dependencies
-> must be
-> [pinned](https://github.com/release-engineering/cachito/blob/master/docs/pip.md#pinning-versions)
-> to an exact version. Cachito will refuse to process requirements files that use the --index-url or
-> --extra-index-url options, which means private registries are out of the question. These two
-> restrictions should eliminate most attack vectors.
->
-> To protect yourself even further, use pip’s
-> [hash-checking mode](https://pip.pypa.io/en/stable/reference/pip_install/#hash-checking-mode). Note
-> that pip does not support hash checking for VCS dependencies, e.g. git. Consider transforming your
-> git dependencies to plain https dependencies. For example, if the repository is hosted on github,
-> you can use https://github.com/{org_name}/{repo_name}/{commit_id}.tar.gz to get the tarball for a
-> specific commit.
->
-> ## npm
->
-> *TL;DR: do not use unofficial registries. Even if you try to do so via .npmrc, Cachito will ignore
-> it.*
->
-> Npm packages follow the typical package file + lock file approach. The package file is
-> [package.json](https://docs.npmjs.com/cli/v6/configuring-npm/package-json), the lock file is
-> [package-lock.json](https://docs.npmjs.com/cli/v6/configuring-npm/package-lock-json) or
-> [npm-shrinkwrap.json](https://docs.npmjs.com/cli/v6/configuring-npm/shrinkwrap-json). Cachito
-> requires the lock file.
->
-> The lock file pins all dependencies to exact versions. For https dependencies, which are impossible
-> to pin, Cachito requires the integrity value (a checksum). For dependencies from the npm registry,
-> Cachito does not require integrity values, but newer versions of npm will always include them. Check
-> if all your dependencies have an integrity value, update your lock file if not.
->
-> Do not try to use private registries. If you point npm to a private registry (or any registry other
-> than the official one) via [.npmrc](https://docs.npmjs.com/cli/v6/configuring-npm/npmrc), Cachito
-> will ignore it and look in the official registry anyway.
```

Comment:
I don't think we've made any distinctive changes to NPM after porting the code from cachito, so I guess it's still valid.